packages/webapp/src/components/Modals/WaterUsageCalculatorModal/index.jsx

```diff
@@ -7,7 +7,7 @@ import Button from '../../Form/Button';
 import PropTypes from 'prop-types';
 import Unit, { getUnitOptionMap } from '../../Form/Unit';
 import {
-  estimated_duration,
+  estimated_duration as estimated_time_duration,
```

Comment:
irrigation_task_estimated_duration
same with other units db_table_column_name

Reply:
@JimmyRowland  this is a unit used by other components. I can't rename it I can as only type it.

Reply:
The convention is to create new ones for each field in each table as allowed units might be different.